### ISSUE TITLE
Creating the Use Case Diagram

### ISSUE THREAD

**Issue by ahmetnecipg** [bounswe/2021SpringGroup2#30]
We have to construct a Use Case Diagram which clearly indicates the actor types(Player/Spectator, Poster, Guest User) and what they can do.

**Comment by ege-kaya** [comment]
I worked mainly on the guest user section of the diagram, and generally made small improvements to the diagram

**Comment by berkaydoner** [comment]
I listed the main functionalities accessible by the actors and helped with the overall structure of the diagram.

**Comment by ahmetnecipg** [comment]
I worked heavily on the Player/Spectator part of the diagram mainly on "View Own Profile" , "User Search" and "Event Search" cases. I also worked on figuring out the 'extend, include' arrows. After the diagram was complete I divided it into parts, screen snipped them and put them together in mspaint so we could have a legible diagram we can put on our github repository. The final form of the diagram : https://github.com/bounswe/2021SpringGroup2/wiki/Use-Case-Diagram
